### ISSUE TITLE
Replace publications_export_template with script 

### ISSUE THREAD

**Issue by lobodemonte** [NYU-CI/RichContextMetadata#24]
Instead of using a Jupyter notebook to generate the partition json files, it'd be better to use a script that can take either: 
1. folder name, csv file name to be exported 
2. file with rows of folder names and csv file names to be exported 

Need to make it OS insensitive

**Comment by ceteri** [comment]
I agree! That way we develop a library of functions, and have them tested for how they handle edge cases on each commit.
